service/client/client.js

```diff
@@ -110,8 +110,8 @@ export default class Client {
         return `${this.url}${this.urlVersion}/users`;
     }
 
-    getFilesRoute(teamId) {
-        return `${this.url}${this.urlVersion}/teams/${teamId}/files`;
+    getFilesRoute() {
```

Comment:
If that's the case I think you need to go ahead and do that on this PR as that one I think is used in a test somewhere, can you double check?